tests/test_cross_validation.cc

```diff
@@ -42,7 +42,7 @@ TEST(test_cross_validation, test_fold_creation) {
 
 bool is_monotonic_increasing(const Eigen::VectorXd &x) {
   for (Eigen::Index i = 0; i < x.size() - 1; i++) {
-    if (x[i + 1] - x[i] <= 0.) {
+    if (x[i + 1] - x[i] < 0.) {
```

Comment:
The definition of monotonic increasing is just that it doesn't decrease, right? Perhaps rename it if we want this change.

Reply:
I think this actually matches the definition now.  Before it was checking that it "always increases" but as you pointed out monotonic increasing just means "doesn't decrease" which is what it now checks.

Reply:
Oh, my mistake. :)